Gemfile

```
@@ -7,6 +7,9 @@ gem "github-linguist", "~> 7.24"
 gem "jekyll", "~> 4.3"
 
 group :jekyll_plugins do
+  gem "github-pages"
+  gem "jekyll-feed"
+  gem "jekyll-sitemap"
   gem "jekyll-redirect-from"
   gem "jekyll-relative-links"
 end
```

```suggestion
  gem "github-pages"
end
```

Refs: https://pages.github.com/versions/